Site chores, and fixes

SBTopZZZ-LG opened the pull request:
1. Update Sumit Naik's picture.
2. Update the site's metadata, basically the title, favicon and other icons.
3. Fixed the horizontal overflow issue on the homepage.
4. Update email addresses in the backend's `configs/meta.json`.
5. Update references to Aura 23 in Policies with Aura 24.
6. Fixed an issue where team registration would fail as the validation considers the previous year's Aura ID matcher.
7. Update my picture with a non-cringe one, and headline to "Full Stack Developer".
8. Fixed outdated registration fee scheme to account for the below:

| Teams of | Reg. fees |
|----------|-----------|
|Solo|Rs. 99|
|Duet|Rs. 199|
|2-4|Rs. 299|
|4+|Rs. 399|

9. Fixed a major issue where the logic to find the team of a user fails in certain edge cases, due to the API response being paginated.

SBTopZZZ-LG committed:
chore: update Sumit Naik's picture

SBTopZZZ-LG committed:
chore: update site metadata, icons, and favicon

SBTopZZZ-LG committed:
fix: horizontal overflow issue on homepage

SBTopZZZ-LG committed:
chore: update email addresses in meta.json

SBTopZZZ-LG committed:
refactor: update old names and emails in the policies

SBTopZZZ-LG committed:
fix: incorrect aura id matching that will fail registration

SBTopZZZ-LG committed:
refactor: update my picture to a non-cringe one

SBTopZZZ-LG committed:
chore: update headline to full stack dev 💯

sudeepk11 commented:
Thanks for the description

sudeepk11 commented on backend/configs/meta.json:
```diff
@@ -3,8 +3,8 @@
   "dev_host": "http://localhost:3000",
   "name": "Aura 24",
   "emails": {
-    "primary": "aura23.klsgit@gmail.com",
```
Pick this up from the env so that any changes we make due to the email limit is reflected here without the code being changed.

SBTopZZZ-LG replied:
![](https://github.com/sudeepk11/aura-git/assets/25717503/7c9a92b6-8670-45f4-8022-24899fcc0e1d)

The usages are only applicable for specifying an email address to reply to, in the emails. So in short these are not used by the nodemailer in any way, so no worries.

SBTopZZZ-LG committed:
fix: outdated registration fee scheme

SBTopZZZ-LG committed:
fix: incorrect registration fee displayed after registering for an event

SBTopZZZ-LG committed:
fix: incorrect logic for fetching all teams containing a user

The logic breaks in cases where the user
is registered in many events, and because
the API returns a paginated response of teams,
the logic fails to find the appropriate team and
does not take into account the edge cases.
